JSDemos/Demos/Scheduler/CustomTemplates/description.md

```diff
@@ -1 +1,7 @@
-This demo illustrates the rich collection of&nbsp;the appearance customization properties available as&nbsp;a&nbsp;part of&nbsp;the Scheduler component. The appearance of&nbsp;appointments and appointment tooltips are defined using custom templates. Properties for the appointment details form are set within the **appointmentFormOpening** event handler. Properties for the Scheduler are also modified: the **timelineDay** view is&nbsp;added to&nbsp;the _View Selector_ and the _All-day panel_ is&nbsp;hidden.
\ No newline at end of file
+This demo shows the following Scheduler properties that specify custom templates:
+
+* [appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTemplate) / **views[]**.[appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTemplate)
```

Comment:
Why did you decide to omit the descriptions? How can users understand the difference between these properties? 

Also, could you please show in which line we demonstrate the **views[]**.appointementTemplate property? I guess that it should be used to customize an individual view but I can't see it anywhere. The same notes apply to the next list item.